README.md

```diff
@@ -48,6 +48,48 @@ To run tests via Docker:
 docker-compose run mozilla/fxa_customs_server npm test
 ```
 
+## Tagging Releases
+
+Unlike other FxA services,
+the customs-server includes some non-public code changes
+that are managed in a separate private repo:
+
+  https://github.com/mozilla/fxa-customs-server-private/
+
+When tagging a new release for deployment,
+it needs to be merged to the private repo
+and re-tagged with those changes in place.
+The process looks something like this:
+
+```
+> # First, make a new public tag.
+> cd ./fxa-customs-server
+> grunt version
+> git push; git push --tags
+>
+> # Next, merge the updates to the private repo.
+> cd ../fxa-customs-server-private
+> git checkout master ; git pull
+> git remote add public https://github.com/mozilla/fxa-customs-server
+> git fetch public
+> git merge public/master
+> git push
+>
+> # Make a release branch in the private repo.
+> git checkout v1.XXX.0
+> git checkout -b train-XXX
+> git push -u origin train-XXX
+>
+> # Merge private changes from previous train.
+> git merge origin/train-(XXX-1)
+>
+> # Make a private tag to include those changes.
+> git tag v1.XXX.0-private
+> git push; git push --tags
```

Comment:
I wonder if some super rare git configurations or a bad origin setup this might push a private tag to a public repo by accident, but i might be over cautious

Reply:
Yeah, I recommend always checking out the private repo separately so that it's `origin`, I've had a few accidents when trying to manage it as a secondary remote in an otherwise public repo :-(

Reply:
I've had those accidents before too and never use `git push --tags` any more, I always push the specific tag I want explicitly. But I am quite clumsy, so maybe that level of precaution is not for everyone.

Reply:
This sounds like a reasonable precaution FWIW

Reply:
I have a script that I use to do this:

> git push origin refs/tags/$TAG_NAME --no-verify

That avoid pushing unexpected tags.